client/pkg/apigee/proxy.go

```diff
@@ -65,6 +82,50 @@ func (a *ApigeeClient) GetRevision(proxyName, revision string) (*models.ApiProxy
 	return proxyRevision, nil
 }
 
+// GetRevisionBundle - get a revision bundle of a proxy with a name
+func (a *ApigeeClient) GetRevisionConnectionType(proxyName, revision string) (*HTTPProxyConnection, error) {
+	response, err := a.newRequest(http.MethodGet, fmt.Sprintf("%s/apis/%s/revisions/%s", a.orgURL, proxyName, revision),
+		WithDefaultHeaders(),
+		WithQueryParam("format", "bundle"),
+	).Execute()
+	if err != nil {
+		return nil, err
+	}
+
+	// response is a zip file, lets
+	zipReader, err := zip.NewReader(bytes.NewReader(response.Body), int64(len(response.Body)))
+	if err != nil {
+		return nil, err
+	}
+
+	// Read all the files from the zip archive
+	var fileBytes []byte
+	for _, zipFile := range zipReader.File {
+		if zipFile.Name != "apiproxy/proxies/default.xml" {
+			continue
+		}
+		fileBytes, err = readZipFile(zipFile)
+		if err != nil {
+			return nil, err
+		}
+		break
+	}
+
```

Comment:
do you wanna check in case none of the files were "apiproxy/proxies/default.xml"

Reply:
good call, I will return an error if the file is not found